pkg/bindings/play/play.go

```diff
@@ -59,23 +65,27 @@ func Kube(ctx context.Context, path string, options *KubeOptions) (*entities.Pla
 	return &report, nil
 }
 
-func KubeDown(ctx context.Context, path string) (*entities.PlayKubeReport, error) {
+func KubeDown(ctx context.Context, path string, body io.Reader) (*entities.PlayKubeReport, error) {
```

Comment:
This is a breaking API change. Bindings are stable so we cannot break existing code.

I think it would make more sense to add a new function and call it KubeDownWithBody() and KubeWithBody() and then the old function just opens the path and calls the new function. This would allow your usecase and old code to function properly.

Reply:
OK - Done.